Ugizwenayo-Divine/FoodStuffPriceCompare-App
# Setting up sequelize ORM

#### What does this PR do?
- setting up the sequelize ORM
#### Description of Task to be completed?
- created .sequelizerc file
- installed all required node modules
- created the sequelize folder structure by sequelize init
- added required environment variables
- created the database
- created the model with migrations
- created the tables by npx sequelize-cli db:migrate
#### How should this be manually tested?
- After cloning the project repo, run npm i inside the cloned repo folder
- change in cloned repo
- git fetch
- git checkout ch-setup-sequelize
- npm i
- instert all required env variables in .env file
- run `npx sequelize-cli model:generate --name model-name --attributes all attributes with datatypes`
- you should see the model with model-name you provided
- run `npx sequelize-cli db:create` you should see the database with the name in .env file
- run `npx sequelize-cli db:migrate` you should see tables for models you have
#### Any background context you want to provide?
- you must have node installed locally
- you must have the postgres installed on your local machine and configured
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
